<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGUIClass</class>
 <widget class="QMainWindow" name="QtGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>851</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="RawVideo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ProcessedVideo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="RedMinSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="RedMaxSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="GreenMinSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>129</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="GreenMaxSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="BlueMaxSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="BlueMinSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="RedMinValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="RedMaxValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="GreenMinValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="GreenMaxValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="BlueMinValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="BlueMaxValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="RedMinLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Red Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="RedMaxLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Red Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="GreenMinLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Green Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="GreenMaxLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Green Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="BlueMinLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Blue Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="BlueMaxLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Blue Max</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="LoopValue">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>420</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PauseButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="LoopLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>400</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Loop Time (ms)</string>
    </property>
   </widget>
   <widget class="QLabel" name="FPSLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>470</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="FPSValue">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>490</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="GestureIDLabel">
    <property name="geometry">
     <rect>
      <x>1056</x>
      <y>412</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gesture ID</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="GestureIDValue">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>400</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="GestureCoordValue">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>448</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="GestureCoordLabel">
    <property name="geometry">
     <rect>
      <x>1036</x>
      <y>460</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Gesture Coords</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="CursorCoordValue">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>498</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="CursorCoordLabel">
    <property name="geometry">
     <rect>
      <x>1036</x>
      <y>510</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cursor Coords</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGUI.qrc"/>
 </resources>
 <connections/>
</ui>
